consensus/state.go

```diff
@@ -2267,9 +2267,10 @@ func (cs *State) signVote(
 func (cs *State) voteTime() time.Time {
 	now := tmtime.Now()
 	minVoteTime := now
+	// Minimum time increment between blocks
+	timeIota := time.Millisecond
```

Comment:
```suggestion
	const timeIota = time.Millisecond
```
doesn't and shouldn't change.